app/actions/CustomActionBuilders.scala

```diff
@@ -31,30 +31,36 @@ class CustomActionBuilders(
 
   import CustomActionBuilders._
 
+  val membersIdentityClientId = "clientId" -> "members"
+
+  val recurringIdentityClientId = "clientId" -> "recurringContributions"
+
   // Tells identity to send users back to the checkout immediately after sign-up.
   private val idSkipConfirmation: (String, String) = "skipConfirmation" -> "true"
   // Prevents the identity validation email sending users back to our checkout.
   private val idSkipValidationReturn: (String, String) = "skipValidationReturn" -> "true"
 
-  private val idMember = "clientId" -> "members"
-
-  private def idWebAppRegisterUrl(path: String): String =
-    idWebAppUrl / "register" ? ("returnUrl" -> s"$supportUrl$path") & idSkipConfirmation & idSkipValidationReturn & idMember
+  private def idWebAppRegisterUrl(path: String, clientId: (String, String)): String =
+    idWebAppUrl / "register" ? ("returnUrl" -> s"$supportUrl$path") & idSkipConfirmation & idSkipValidationReturn & clientId
 
-  private def newSignInFlowIdWebAppRegisterUrl(path: String): String =
-    idWebAppUrl / "signin/start" ? ("returnUrl" -> s"$supportUrl$path") & idSkipConfirmation & idSkipValidationReturn & idMember
+  private def newSignInFlowIdWebAppRegisterUrl(path: String, clientId: (String, String)): String =
```

Comment:
Following discussion with @Ap0c - not sure this is what you intended. You're passing `membersIdentityClientId` to the new sign-in flow and not using your `clientId` param. But the clientId used by the new flow is "recurringContributions" so this won't give us the custom copy we want

Reply:
Thinking about this again, given that the new clientId only applies to the new sign-in flow that we're testing, it seems like it makes more sense just to hardcode it in this function and not touch anything else. Because if the test doesn't do well, we'll presumably be removing this code anyway

Reply:
This was supposed to be `clientId`, see the comment above. I'd prefer to keep it in the same style as `idWebAppRegisterUrl` personally.

Reply:
Thinking about this more, this probably doesn't need to be a separate function, as it is so similar to the exisiting one. 

I've just changed it to this: 
```
  private def idWebAppUrl(path: String, clientId: (String, String)): String =
    idWebAppUrl ? ("returnUrl" -> s"$supportUrl$path") & idSkipConfirmation & idSkipValidationReturn & clientId 
```

And passed either `request.uri / "register" ` or `request.uri / "signin/start"` to it depending on where it's coming from.

Reply:
I've thought about this more and rather than have seperate functions for `/register` and `/signin/start`, I've sent the path in as a parameter, see below:

```
private def idWebAppUrl(path: String, clientId: (String, String), idWebAppPath: String): String =
    idWebAppDomain / idWebAppPath ? ("returnUrl" -> s"$supportUrl$path") & idSkipConfirmation & idSkipValidationReturn & clientId
```